src/translations/en.json

```
@@ -1854,6 +1854,7 @@
               "filter": "Filter",
               "show_disabled": "Show disabled entities",
               "show_unavailable": "Show unavailable entities",
+              "show_with_area": "Show etities with area configured",
```

```suggestion
              "show_with_area": "Show entities with area configured",
```
Or perhaps even better "Show entities with assigned area". Reason: The area is not itself configured here, but just assigned / set, so "configured" sounds weird to me here.

---

Agree!